charts/stable/scalardl/templates/ledger/service.yaml

```diff
@@ -4,9 +4,16 @@ metadata:
   name: {{ include "scalardl.fullname" . }}-ledger-headless
   labels:
     {{- include "scalardl-ledger.labels" . | nindent 4 }}
+  annotations:
+{{ toYaml .Values.ledger.service.annotations | indent 4 }}
```

Comment:
Looking good! But I'm not sure what's the best.

- envoy
https://github.com/scalar-labs/scalar-kubernetes/blob/a05a5a679a2590a138a24c310b4c8d15ea13cc0e/charts/stable/scalardl/templates/envoy/service.yaml#L7-L8

Maybe this is the best.
```
  annotations:
    {{- toYaml .Values.ledger.service.annotations | nindent 4 }}
```

https://github.com/helm/charts/blob/2f030a8e9db3e451e8030b12d2658371f5708f7d/stable/prometheus-mongodb-exporter/templates/deployment.yaml#L10-L11

Reply:
It looks good because it matches the `label:` section on the previous line. https://github.com/scalar-labs/scalar-kubernetes/pull/106/files#diff-3a191c69fc28a153725a3ebc042dba5fe5b849a49fb1244817ac71a7d5601d35R5-R6

Reply:
@tei-k Updating it? or Is it ready to be merged?

Reply:
Updated in 7ff2b82. 🙏